pkg/service/roomallocator.go

```diff
@@ -68,6 +68,12 @@ func (r *StandardRoomAllocator) CreateRoom(ctx context.Context, req *livekit.Cre
 	if req.Metadata != "" {
 		rm.Metadata = req.Metadata
 	}
+	if req.Egress != nil && req.Egress.Tracks != nil {
+		rm.Egress = &livekit.AutoEgress{
```

Comment:
should we store information about Room auto egress here? even tho it might not be used later on.

Reply:
related, it seems that this information should *not* be returned to clients. it may contain keys/etc that should be server-only. so we'll need some sort of Room.ToProto function that explicitly filters this information out.